<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qgisDemoClass</class>
 <widget class="QMainWindow" name="qgisDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>多光谱影像分类系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget_mapCavans" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actionaddVectorLayer"/>
    <addaction name="actionaddRasterLaer"/>
    <addaction name="actionlayerProperty"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>影像分类</string>
    </property>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>特征提取</string>
    </property>
    <addaction name="actionMoravec"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>影像匹配</string>
    </property>
    <addaction name="actionpipei"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionaddVectorLayer">
   <property name="text">
    <string>添加矢量图层</string>
   </property>
  </action>
  <action name="actionaddRasterLaer">
   <property name="text">
    <string>添加栅格图层</string>
   </property>
  </action>
  <action name="actionlayerProperty">
   <property name="text">
    <string>属性表</string>
   </property>
  </action>
  <action name="actionstyleManager">
   <property name="text">
    <string>风格管理器</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>影像分类控制窗口</string>
   </property>
  </action>
  <action name="actionMoravec">
   <property name="text">
    <string>Moravec</string>
   </property>
  </action>
  <action name="actionpipei">
   <property name="text">
    <string>影响匹配窗口</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qgisDemo.qrc"/>
 </resources>
 <connections/>
</ui>
